.github/workflows/asan.yaml

```diff
@@ -2,20 +2,21 @@ name: asan
 on:
   pull_request:
   push:
+    branches: [ main ]
   schedule:
     - cron: '7 0 * * *'
 
 jobs:
   asan:
     name: asan
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Knit: I have no idea why Im Kay-pohing this, but wouldn't it be better to lock onto ubuntu-22.04 instead? Seems availabel here: https://github.com/actions/runner-images/issues/5490 . Otherwise we may end up breaking our CI once ubuntu22.10 comes out.

Reply:
From what I understand the ubuntu-latest here would only be the "base" on which the container specified below runs, so as long as it has a functional Docker implementation it should be fine. They also only seem to stick to LTS releases for the `latest` tag so I guess it should be OK?